Source/Core/Common/Logging/LogManager.cpp

```diff
@@ -87,6 +87,7 @@ LogManager::LogManager()
   m_Log[LogTypes::SP1] = new LogContainer("SP1", "Serial Port 1");
   m_Log[LogTypes::VIDEO] = new LogContainer("Video", "Video Backend");
   m_Log[LogTypes::VIDEOINTERFACE] = new LogContainer("VI", "Video Interface (VI)");
+  m_Log[LogTypes::VIDEOCUSTOMTEX] = new LogContainer("VCT", "Video Custom Textures (VCT)");
```

Comment:
For log category names, we only add acronyms in parentheses if the acronym is likely to be easier to recognize than the full name. That's definitely not the case for the acronym you're adding - I've never seen it be used even a single time before. Please remove it.